docs/pages/desktop-access/rbac.mdx

```diff
@@ -67,14 +67,19 @@ label to a Windows desktop named `test.dev.example.com` and the
 
 ```yaml
 host_labels:
-  - match: '^.*\.dev\.example\.com$'
+  - match: '^.*\.dev\.example\.com'
     labels:
       environment: dev
-  - match: '^.*\.prod\.example\.com$'
+  - match: '^.*\.prod\.example\.com'
     labels:
       environment: prod
 ```
 
+The regular expression provided in the `match` rule above is applied to the desktop's
```

Comment:
I think it would help to include examples for:
- A desktop discovered via LDAP
- A host that is statically defined

Reply:
Updated, let me know if you think that's an improvement.